INFO     root:api.py:98 Invalid event_history_id 2e730156-a55c-4893-afa2-e3fce8aab925 provided